include/aspect/postprocess/matrix_statistics.h

```diff
@@ -0,0 +1,61 @@
+/*
+  Copyright (C) 2016 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef __aspect__postprocess_matrix_statistics_h
+#define __aspect__postprocess_matrix_statistics_h
+
+#include <aspect/postprocess/interface.h>
+#include <aspect/simulator_access.h>
+
+namespace
+{
+  const std::string
+  get_stats(const aspect::TrilinosWrappers::BlockSparseMatrix &matrix,
+            const std::string matrix_name,
+            const MPI_Comm &comm);
```

Comment:
There isn't actually a need to forward declare this function. In fact, forward declaring things in anonymous namespaces has no useful effect because every other `.cc` file that may `#include` this one would see this function in a different namespace than where it really is (because every `.cc` has a unique anonymous namespace). Just remove the declaration.